src/gallium/drivers/lima/lima_draw.c

```diff
@@ -353,7 +353,7 @@ lima_pack_plbu_cmd(struct lima_context *ctx, const struct pipe_draw_info *info)
       ctx->buffer_state[lima_ctx_buff_pp_plb_rsw].size;
    plbu_cmd[i++] = 0x80000000 | (gl_position_va >> 4); /* RSW_VERTEX_ARRAY */
 
-   if (ctx->dirty & LIMA_CONTEXT_DIRTY_SCISSOR) {
+   if (ctx->rasterizer->base.scissor) {
```

Comment:
This makes every draw PLBU cmd has the scissor part if scissor is enabled. It seems to me that we just need the first draw that enabled the scissor to have this part and latter draw need not it if scissor is not dirty. Does your binary dump show the same thing?